Patch-List-Repo/src/main/java/org/wso2/patchvalidator/service/Signer.java

```diff
@@ -50,7 +50,7 @@ static StringBuilder sign() {
         }
 
         if (patchesList.isEmpty()) {
-            LOG.info("No patches in the "Ready to sign" state");
+            LOG.info("No patches in the Ready to sign state");
```

Comment:
```suggestion
            LOG.info("No patches in the `Ready to sign` state");
```
Can we add something like this?